<launch>
    <arg name="feed_path" default="/camera/usb/front/compressed" />
    <arg name="model" default="2024_gate_glyphs" />
    <arg name="rgb_raw" default="True" />
    <arg name="rgb_detections" default="True" />
    <arg name="depth" default="False" />
    <arg name="sync_nn" default="True" />
    <arg name="using_sonar" default="False" />
    <arg name="show_class_name" default="True" />
    <arg name="show_confidence" default="True" />
    <node name="depthai_mono_detection" pkg="cv" exec="depthai_mono_detection" output="screen">
        <param name="feed_path" value="$(var feed_path)" />
        <param name="model" value="$(var model)" />
        <param name="rgb_raw" value="$(var rgb_raw)" />
        <param name="rgb_detections" value="$(var rgb_detections)" />
        <param name="depth" value="$(var depth)" />
        <param name="sync_nn" value="$(var sync_nn)" />
        <param name="using_sonar" value="$(var using_sonar)" />
        <param name="show_class_name" value="$(var show_class_name)" />
        <param name="show_confidence" value="$(var show_confidence)" />
    </node>
</launch>
